### PR TITLE
History

### PR DESCRIPTION
**Pull request by SadiqZak**
Added History functionality that checks to see videos visited

### CODE REVIEW

**Comment by SadiqZak** [comment]
history

File: src/utils/reducer.js [hunk 1]
```diff
@@ -0,0 +1,290 @@
+const reducerFunc = (state, action) => {
+  switch (action.type) {
+    case "initialState":
+      // const handleLiked =()=>{
```
Comment (rahulbakshi4) [inline src/utils/reducer.js]: Remove the commented code

File: src/utils/reducer.js [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+        ...state,
+        playlists:[],
+        videoLibUpdated: clearState,
+      }
+    case "WatchHistory":
+      const historyUpdate=()=>{
+
+        const updatedHistoryList = state.history.find((item)=>item.id===action.payload.id)
+
+        console.log(updatedHistoryList)
```
Comment (rahulbakshi4) [inline src/utils/reducer.js]: Remove console logs

**Review by rahulbakshi4 — APPROVED** [review]
Code Structure is good
Functionalities are working

Suggestion : 
- Made some comments look into it
- Keep code in PR short for better reviews and code related to single functionality

**Review by avindhawale — APPROVED** [review]
Good code structure, good use of css.
Suggestion: 
- While writing responsive css, consider mobile first approach.

**Pull request merged**